src/Raven.Server/Config/Categories/CoreConfiguration.cs

```diff
@@ -22,6 +22,11 @@ public class CoreConfiguration : ConfigurationCategory
         [ConfigurationEntry("ServerUrl.Tcp", ConfigurationEntryScope.ServerWideOnly)]
         public string[] TcpServerUrls { get; set; }
```

Comment:
Why do we need this and `TcpPriorityServerUrls` ? 
Can we have just one such configuration? What happens if we provide multiple ips here?

Reply:
That is what I understood the request was if not then the issue should be more clear.
TcpServerUrls are automatically generated and populated and have validations that there is no more than one item there for some cases.
I found it wrong to use it.
We want to have multiple ips that is the whole point of this code, that server could have a priority list of ips they can connect to and it contains the private tcp url by the order of which you prefer to connect.
This is what i did now the server->server communication is using those ips to open a tcp client and not the public tcp url.